test-suite/shared/src/test/scala/org/scalajs/testsuite/niocharset/UTF8Test.scala

```diff
@@ -16,26 +16,36 @@ import org.junit.Assert._
 import BaseCharsetTest._
 
 class UTF8Test extends BaseCharsetTest(Charset.forName("UTF-8")) {
-  @Test def decode(): Unit = {
-    def OutSeq(elems: OutPart[CharBuffer]*): Seq[OutPart[CharBuffer]] =
-      Seq[OutPart[CharBuffer]](elems: _*)
-
+  @Test def decode1byte(): Unit = {
```

Comment:
I grouped up the decode tests, in order to more easily identify test-cases. Happy to undo again if that's your preference, though this might be easier for the next person coming through.